cmd/prometheus/main.go

```diff
@@ -634,28 +636,28 @@ func main() {
 		mode = "agent"
 	}
 
-	level.Info(logger).Log("msg", "Starting "+modeAppName, "mode", mode, "version", version.Info())
+	logger.Info("Starting "+modeAppName, "mode", mode, "version", version.Info())
 	if bits.UintSize < 64 {
-		level.Warn(logger).Log("msg", "This Prometheus binary has not been compiled for a 64-bit architecture. Due to virtual memory constraints of 32-bit systems, it is highly recommended to switch to a 64-bit binary of Prometheus.", "GOARCH", runtime.GOARCH)
+		logger.Warn("This Prometheus binary has not been compiled for a 64-bit architecture. Due to virtual memory constraints of 32-bit systems, it is highly recommended to switch to a 64-bit binary of Prometheus.", "GOARCH", runtime.GOARCH)
 	}
 
-	level.Info(logger).Log("build_context", version.BuildContext())
-	level.Info(logger).Log("host_details", prom_runtime.Uname())
-	level.Info(logger).Log("fd_limits", prom_runtime.FdLimits())
-	level.Info(logger).Log("vm_limits", prom_runtime.VMLimits())
+	logger.Info(version.BuildContext())
+	logger.Info(prom_runtime.Uname())
+	logger.Info(prom_runtime.FdLimits())
+	logger.Info(prom_runtime.VMLimits())
```

Comment:
I think we should retain the names here, otherwise it is hard to interpret the output.

Reply:
I'll agree with that. They were trimmed initially to fit the output into slog's pre-existing `msg` key. 

Would it be better to log them all as keyvals in a single log call? 

```go
	logger.Info("operational information",
		"build_context", version.BuildContext(),
		"host_details", prom_runtime.Uname(),
		"fd_limits", prom_runtime.FdLimits(),
		"vm_limits", prom_runtime.VMLimits(),
	)
```

Reply:
updated to a single log entry with keyvals, can change further is desired